app/controllers/api/v1/cocktails_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::CocktailsController < ApplicationController
+    before_action :check_query_param
+
+    def index
+        response = RestClient.get(
+            "https://www.thecocktaildb.com/api/json/v1/1/search.php",
+            {
+                params: {
+                    s: params[:q]
```

Comment:
I don't know if this is out of scope for this PR, but it doesn't seem like we are sanitizing the data but pulling back all data for the drinks in the results. Is this something that would be relevant to include in this PR?

Reply:
Fixed it with Aditya